src/main/java/com/epam/model/Person.java

```diff
@@ -7,7 +7,7 @@
 @Data
 public class Person {
     private Integer id;
-    private String name;
+    private String email;
```

Comment:
Can you merge current dev in your branch and track, that this PR contains only reservation stuff changes?